htdocs/install/mysql/migration/15.0.0-16.0.0.sql

```diff
@@ -279,4 +279,11 @@ ALTER TABLE llx_propal ADD last_main_doc VARCHAR(255) NULL AFTER model_pdf;
 
 UPDATE llx_c_country SET eec=0 WHERE eec IS NULL;
 ALTER TABLE llx_c_country MODIFY COLUMN eec tinyint DEFAULT 0 NOT NULL;
-
+ALTER TABLE llx_c_availability ADD COLUMN type_duration char(1);
```

Comment:
Do not use 'char', use always varchar, even for 1 char.

Reply:
i did it